### PR TITLE
Add [source,terminal] to CLI commands

### PR DESCRIPTION
**Pull request by apinnick**
CP for 4.5, 4.6

### CODE REVIEW

**Comment by apinnick** [comment]
@aburdenthehand  Would you mind reviewing this? Thanks

File: modules/migration-configuring-gcp.adoc [hunk 1]
```diff
@@ -21,6 +21,7 @@ You can configure a Google Cloud Provider (GCP) storage bucket as a replication
 
 . Run `gsutil init` to log in:
 +
+[source,terminal]
 ----
 $ gsutil init
 Welcome! This command will take you through the configuration of gcloud.
```
Comment (aburdenthehand) [inline modules/migration-configuring-gcp.adoc:27]: This output should be detached from the command and inc the .Example output title and terminal metadata ie:
[source,terminal]
----
$ gsutil init
----
+
.Example output
[source,terminal]
----
Welcome! This ... etc

File: modules/migration-installing-cam-operator-ocp-3.adoc [hunk 2]
```diff
@@ -94,19 +99,22 @@ endif::[]
...
 . Create the Cluster Application Migration Operator CR object:
 +
+[source,terminal]
 ----
 $ oc create -f operator.yml
 ----
 +
 The output resembles the following:
 +
+[source,terminal]
```
Comment (aburdenthehand) [inline modules/migration-installing-cam-operator-ocp-3.adoc:117]: Needs the "Example output" title

File: modules/migration-restic-errors.adoc [hunk 3]
```diff
@@ -38,6 +38,7 @@ spec:
 
 If data verification fails when migrating a PV with the filesystem data copy method, the following error appears in the MigMigration Custom Resource (CR):
 
+[source,yaml]
```
Comment (aburdenthehand) [inline modules/migration-restic-errors.adoc:41]: "Example output" title

File: modules/migration-restic-errors.adoc [hunk 4]
```diff
@@ -64,12 +65,14 @@ You can check the target cluster's Restore CR to identify the source of the data
...
 . View the Restore CR:
 +
+[source,terminal]
 ----
 $ oc describe <registry-example-migration-rvwcm> -n openshift-migration
 ----
 +
 The output identifies the PV with `PodVolumeRestore` errors:
 +
+[source,yaml]
```
Comment (aburdenthehand) [inline modules/migration-restic-errors.adoc:75]: "Example output" title

File: modules/migration-restic-errors.adoc [hunk 5]
```diff
@@ -85,12 +88,14 @@ status:
...
 . View the `PodVolumeRestore` CR:
 +
+[source,terminal]
 ----
 $ oc describe <migration-example-rvwcm-98t49>
 ----
 +
 The output identifies the Restic Pod that logged the errors:
 +
+[source,yaml]
```
Comment (aburdenthehand) [inline modules/migration-restic-errors.adoc:98]: "Example output" title

File: modules/migration-viewing-migration-crs.adoc [hunk 6]
```diff
@@ -45,20 +45,23 @@ image:darkcircle-8.png[20,20] link:https://github.com/heptio/velero/blob/master/
...
+[source,terminal]
 ----
 $ oc get <migration_cr> -n openshift-migration <1>
 ----
 <1> Specify the migration CR, for example, `migmigration`.
 +
 The output is similar to the following:
-+
+
+[source,terminal]
```
Comment (aburdenthehand) [inline modules/migration-viewing-migration-crs.adoc:56]: "Example output" title

**Pull request merged**